# Inputs
string table_name
string entry_name
bool[] default_value
---
# Outputs
bool[] output
time last_valid